cluster-scope/base/namespaces/opendatahub-operator/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: opendatahub-operator
+
+resources:
+  - namespace.yaml
+
+components:
+  - ../../../components/project-admin-rolebindings/operate-first
```

Comment:
let's remove the operate-first group from this? since this would let anyone in the group become cluster-admin